Sources/tart/VM.swift

```diff
@@ -383,13 +385,19 @@ class VM: NSObject, VZVirtualMachineDelegate, ObservableObject {
     }
 
     // Storage
-    var attachment = try VZDiskImageStorageDeviceAttachment(
```

Comment:
Would it be possible to just revert this part of code?

It doesn't need any changes.